rust/chains/hyperlane-ethereum/src/interchain_gas.rs

```diff
@@ -130,6 +124,16 @@ where
     }
 }
 
+#[async_trait]
+impl<M> SequenceIndexer<InterchainGasPayment> for EthereumInterchainGasPaymasterIndexer<M>
+where
+    M: Middleware + 'static,
+{
+    async fn nonce_at_tip(&self) -> ChainResult<(u32, u32)> {
+        panic!("Gas payment nonce indexing not implemented");
```

Comment:
Maybe worth a quick comment in each of these impls that panic for future readers explaining why this is okay to panic here / why we need to implement this trait even if it's not used

Reply:
Don't know why this change doesn't show up in the PR view, if you check my branch this has already been changed :/